README.md

```diff
@@ -1,21 +1,88 @@
-# Website
+# Get Started
 
 This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 
-## Installation
+Docusaurus is a **static-site generator**. It builds a **single-page application** with fast client-side navigation, leveraging the full power of **React** to make your site interactive. It provides out-of-the-box **documentation features** but can be used to create **any kind of site** (personal website, product, blog, marketing landing pages, etc).
+
+## Dependencies
+
+* **Nodejs** version >= 14 or above
+
+* **Yarn** version >= 1.5
+
+* **NPM** [Guide](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+
+## How to install
+
+## Install Dependencies for Windows users
+
+Download [Nodejs installer](https://nodejs.org/en/download/) from their Official website.
+
+### Install yarn
 
-```console
-yarn install
+```
+npm install --global yarn
+```
+
+## Install Dependencies for Mac Users
+
+Download [Nodejs installer](https://nodejs.org/en/download/) from their Official website.
+
+Alternatively:
+
+**Using Homebew**
+
+```
+brew install node
+```
+
+**Using MacPorts**
+
+```
+sudo port install nodejs10
```

Comment:
This installs an old version of node that this project doesnt support. It should be a more recent version.

https://ports.macports.org/search/?q=nodejs&name=on